### ISSUE TITLE
Error about importing #%linea-default-line-macro

### ISSUE THREAD

**Issue by vityou** [willghatch/racket-rash#19]
I get this after updating rash:
```
error in rc file /home/vityou/.config/rash/rashrc.rkt: .config/rash/rashrc.rkt:9:11: module: identifier already imported from: rash/private/basic-module-begin
  at: #%linea-default-line-macro
  in: "aliases.rkt"
  location...:
   .config/rash/rashrc.rkt:9:11
   .config/rash/rashrc.rkt:1:6
  context...:
   standard-module-name-resolver
   /home/vityou/.racket/6.12/pkgs/rash/repl.rkt:126:2: for-loop
   /home/vityou/.racket/6.12/pkgs/rash/repl.rkt:113:0: main
   /home/vityou/.racket/6.12/pkgs/rash/repl.rkt: [running body]
```
I don't define a `#%linea-default-line-macro` in my aliases.rkt file:
```
#lang rash

(require linea/defaults linea/line-macro
         (for-syntax racket/base syntax/parse))

(provide (all-defined-out))


; allows defining aliases in the form:
; alias x = y
; or:
; alias x y

(define-line-macro alias
  (syntax-parser
    [(_ i (~datum =) value ...) #'(define-simple-pipeline-alias i value ...)]
    [(_ i value ...) #'(define-simple-pipeline-alias i value ...)]))


; ls aliases
alias ls = 'ls_extended
alias la = ls -a
alias ll = ls -la
alias l = ls -l


(define-line-macro quit
  (syntax-parser
    [(_) #'(exit)]))
```

**Comment by vityou** [comment]
Yes, my rashrc.rkt requires `aliases.rkt`, this is what I'm doing to avoid the error, the commented code produces the error:
```
#lang rash

(require reprovide/reprovide) ; requires and provides all defined out

(provide (all-defined-out))


(reprovide (for-syntax racket/base syntax/parse))

;(reprovide "aliases.rkt" 
;           "line-macros.rkt")
(reprovide (except-in "aliases.rkt" #%linea-default-line-macro))
(reprovide (except-in "line-macros.rkt" #%linea-default-line-macro))
```

**Comment by willghatch** [comment]
I still can't reproduce it.  Would you update again?  The error is related to something I was fixing today, and I wonder if the package server picked up an intermediate commit that was broken or something.

**Comment by willghatch** [comment]
I just updated my non-development install, and I haven't reproduced it on either one.

**Comment by willghatch** [comment]
Also, FYI, I'm going out of town for a week starting tomorrow, so I probably won't be very active here after tonight.

**Comment by willghatch** [comment]
But I'll be in intermittent contact, and maybe I'll fix small bugs.

**Comment by vityou** [comment]
Hmm, after trying to isolate the error, I tried reverting to the file I posted originally, and now I can't reproduce it.  I don't remember updating after I got the error, and I don't remember what exactly I changed.  But anyway, it looks like everything is working, but I'll see if I can reproduce it.

> Also, FYI, I'm going out of town for a week starting tomorrow, so I probably won't be very active here after tonight

Ok, maybe I can use that time to read the code so I can contribute to rash more.  It's already a lot better than bash I think.
